### PR TITLE
Update applications-that-can-bypass-wdac.md

### PR DESCRIPTION
**Pull request by threatpointer**
Included dbgsrv.exe to the ruleset that was missing & a known WDAC bypass.

## Why

Updated the missing WDAC policy

## Changes

Included a WDAC policy with dbgsrv.exe
https://fortynorthsecurity.com/blog/how-to-bypass-wdac-with-dbgsrv-exe/

### CODE REVIEW

**Review by Lo236816 — CHANGES_REQUESTED** [review]
_Clone_

**Comment by threatpointer** [comment]
@jgeurten -- please let me know if you have any questions on this one.

File: windows/security/application-security/application-control/windows-defender-application-control/design/applications-that-can-bypass-wdac.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ Unless your use scenarios explicitly require them, Microsoft recommends that you
 - wsl.exe
 - wslconfig.exe
 - wslhost.exe
+- dbgsrv.exe
```
Comment (jgeurten) [inline windows/security/application-security/application-control/windows-defender-application-control/design/applications-that-can-bypass-wdac.md:60]: Can you sort this entry. The list is sorted alphabetically

File: windows/security/application-security/application-control/windows-defender-application-control/design/applications-that-can-bypass-wdac.md [hunk 2]
```diff
@@ -136,6 +137,7 @@ The blocklist policy that follows includes "Allow all" rules for both kernel and
     <Deny ID="ID_DENY_ADDINPROCESS32" FriendlyName="AddInProcess32.exe" FileName="AddInProcess32.exe" MinimumFileVersion="0.0.0.0" MaximumFileVersion="65355.65355.65355.65355" />
     <Deny ID="ID_DENY_ADDINUTIL" FriendlyName="AddInUtil.exe" FileName="AddInUtil.exe" MinimumFileVersion="0.0.0.0" MaximumFileVersion="65355.65355.65355.65355" />
     <Deny ID="ID_DENY_ASPNET" FriendlyName="aspnet_compiler.exe" FileName="aspnet_compiler.exe" MinimumFileVersion="0.0.0.0" MaximumFileVersion="65355.65355.65355.65355" />
+    <Deny ID="ID_DENY_dbgsrv" FriendlyName="dbgsrv.exe" FileName="dbgsrv.exe" MinimumFileVersion="0.0.0.0" MaximumFileVersion="65355.65355.65355.65355" />
```
Comment (jgeurten) [inline windows/security/application-security/application-control/windows-defender-application-control/design/applications-that-can-bypass-wdac.md:140]: Can you please, 

1. Capitalize the ID, i.e. ID_DENY_DBGSRV
2. Sort this entry as well

Comment (jgeurten) [reply windows/security/application-security/application-control/windows-defender-application-control/design/applications-that-can-bypass-wdac.md:140]: It also needs to be added to <FileRulesRef> in the Windows signing scenario like ID_DENY_BASH, and others, are

**Review by jgeurten — CHANGES_REQUESTED** [review]

File: windows/security/application-security/application-control/windows-defender-application-control/design/applications-that-can-bypass-wdac.md (hunk 2 above)
Comment (threatpointer) [reply windows/security/application-security/application-control/windows-defender-application-control/design/applications-that-can-bypass-wdac.md]: Thanks, I have included as suggested.

File: windows/security/application-security/application-control/windows-defender-application-control/design/applications-that-can-bypass-wdac.md (hunk 1 above)
Comment (threatpointer) [reply windows/security/application-security/application-control/windows-defender-application-control/design/applications-that-can-bypass-wdac.md]: done!

**Review by jgeurten — APPROVED** [review]

**Comment by threatpointer** [comment]
@officedocspr4 @aczechowski: I believe @Lo236816 was mistakenly added as a reviewer & don't have permissions to revoke.

**Pull request merged**